tests/spec/s-error.js

```diff
@@ -19,10 +19,36 @@ describe('Error', function () {
                 throw new RangeError(msg);
             } catch (e) {
                 error = e;
-            };
+            }
             expect(error.name).toBe('RangeError');
             expect(error.message).toBe(msg);
             expect(String(error)).toBe(error.name + ': ' + msg);
         });
     });
+
+    describe('should not list ', function () {
```

Comment:
describe and it descriptions should compose like english sentences, but also stand alone. what is this describing? `it message` isn't a sentence. etc.